exec/jdbc-all/pom.xml

```diff
@@ -420,7 +420,6 @@
                <exclude>org/apache/drill/exec/server/rest/**</exclude>
                <exclude>org/apache/drill/exec/rpc/data/**</exclude>
                <exclude>org/apache/drill/exec/rpc/control/**</exclude>
-               <exclude>org/apache/drill/exec/work/**</exclude>
```

Comment:
can you check with @dsbos or rahul if the JDBC test tool runs fine with this change ?